SETUP.md

```diff
@@ -0,0 +1,166 @@
+# Cloning manubot-rootstock to create a new manuscript
+
+The process to create a new Manubot manuscript is a bit challenging, because it requires a few steps that are difficult to automate.
+However, you will only have to perform these steps once for each manuscript.
+These steps should be performed in a terminal, starting in the directory where you want the manuscript folder be created.
+
+## Configuration
+
+First, you must configure two environment variables (`OWNER` and `REPO`).
+These variables specify the GitHub repository for the manuscript (i.e. `https://github.com/OWNER/REPO`).
+
+```sh
+# GitHub account (change from greenelab)
+OWNER=greenelab
+# Repository name (change from manubot-rootstock)
+REPO=manubot-rootstock
+```
+
+The following commands assume you have SSH key access configured for the `OWNER` GitHub account.
+If this is not the case, see GitHub's guidelines for [generating](https://help.github.com/articles/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent/) a new SSH key and [adding it](https://help.github.com/articles/adding-a-new-ssh-key-to-your-github-account/) to your account.
```

Comment:
We could make this step optional, but the user would have to know how to modify some of the git commands below.